### DISCUSSION TITLE
Welcome to klipper-web-control-docker Discussions!

### DISCUSSION THREAD

**Discussion by dimalo** [dimalo/klipper-web-control-docker#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mechnotech** [comment]
Thanks for the work you've done!
The only repository that allowed me to solve my problem. Namely, the main idea is to have only one Fluid for all printers.
So, I connected Flying Bear 5 via USB to the desktop and K1 (with my own clipper and moonraker via local network.
To do this I had to change several config files to be compatible with the new version of moonraker.

If this helps someone solve a similar problem, I can push the code in the feature branch.
